TeamCode/src/main/java/org/firstinspires/ftc/teamcode/components/DriveSystem.java

```diff
@@ -203,6 +243,10 @@ public int  getMinDistanceFromTarget() {
     }
 
     public boolean driveToPosition(int millimeters, Direction direction, double maxPower) {
+        if (!strafeSet) {
```

Comment:
Why not set this in `driveToPositionTicks`, when we check to see if we're strafing?